AK/StringView.h

```diff
@@ -45,6 +45,7 @@ class StringView {
     }
 
     StringView(const ByteBuffer&);
+    explicit StringView(ByteBuffer&&) = delete;
```

Comment:
Nit pick: might be nicer to put it next to the String temporary one instead of in the middle of all the other constructors